django/thunderstore/api/cyberstorm/tests/test_communities.py

```diff
@@ -0,0 +1,264 @@
+from typing import Dict, List, Union
+
+import pytest
+from django.db.models import Count, Sum
+from django.http import HttpRequest
+from django.urls import reverse
+from rest_framework.exceptions import ParseError
+from rest_framework.test import APIClient
+
+from thunderstore.api.cyberstorm.views.community import CommunitiesAPIView
+from thunderstore.cache.enums import CacheBustCondition
+from thunderstore.cache.tasks import invalidate_cache
+from thunderstore.community.consts import PackageListingReviewStatus
+from thunderstore.community.factories import CommunityFactory, PackageListingFactory
+from thunderstore.community.models.community import Community
+from thunderstore.community.models.community_site import CommunitySite
+
+PARAMS_FIXTURE = {
+    "page_size": 1,
+    "page": 1,
+    "ordering": "name",
+}
+
+
+@pytest.fixture(scope="module", autouse=True)
+def clear_pagination_cache():
+    invalidate_cache(CacheBustCondition.any_package_updated)
+
+
+@pytest.mark.django_db
+def test_CommunitiesAPIView_regular_usage(
+    client: APIClient, community_site: CommunitySite
+):
+    response = client.get(
+        "/api/cyberstorm/c/",
+        HTTP_HOST=community_site.site.domain,
+    )
+    c = Community.objects.annotate(
```

Comment:
Does this test the actual implementation any more after changes?

Reply:
Kind of not. I'll change this to test out the fields only and name it accordingly 👍